packages/gdal/meta.yaml

```diff
@@ -0,0 +1,102 @@
+package:
+  name: gdal
+  version: 3.5.1
+
+source:
+  url: https://github.com/OSGeo/gdal/releases/download/v3.5.2/gdal-3.5.2.tar.gz
+  sha256: fbd696e1b2a858fbd2eb3718db16b14ed9ba82521d3578770d480c74fe1146d2
+requirements:
+  host:
+    - libtiff
+    - libproj
+    - libiconv
+    - geos
+  run:
+    - geos
+
+build:
+  sharedlibrary: true
+  script: |
+    export EMSCRIPTEN_SYSROOT=$(em-config CACHE)/sysroot
+    export EMSCRIPTEN_INCLUDE=$EMSCRIPTEN_SYSROOT/include
+    export EMSCRIPTEN_BIN=$EMSCRIPTEN_SYSROOT/bin
+    export EMSCRIPTEN_LIB=$EMSCRIPTEN_SYSROOT/lib/wasm32-emscripten/pic
+
+    mkdir -p dist
+    export DISTDIR=$(pwd)/dist
+
+    embuilder build zlib --pic
+    embuilder build libjpeg --pic
+    embuilder build libpng --pic
+    embuilder build sqlite3 --pic
+
+    mkdir -p build
+    cd build && emcmake cmake .. \
+      -DCMAKE_INSTALL_PREFIX=$WASM_LIBRARY_DIR \
```

Comment:
So many flags...

Reply:
Actually, after #3128, CMake can now search libraries that are in WASM_LIBRARY_DIR automatically. But in order to prevent some unexpected side effects (selecting a static library instead of a shared library, etc...), I just specified all library paths. I have some future plans to clean up all these verbose flags, but I'm in an "it works, so don't touch it" state 👀